web-client/views/upload.jade

```diff
@@ -240,13 +240,16 @@ html
           br
           p(class='sideHeader')  &nbsp; Viewport Size
           input(type='radio' id='boundBoxS' class='boundBoxSize' name='size' value='containerS')
-          label(for='boundBoxS' style='font-weight:normal' class='sideLabel') Small (1104x648px)
+          label(for='boundBoxS' style='font-weight:normal' class='sideLabel') Small
+          small (1104x648px)
```

Comment:
So did this look OK for you? I’m not in a position to fire up a local copy so just wondering.